src/shell.js

```diff
@@ -111,9 +111,11 @@ if (ENVIRONMENT_IS_NODE) {
 
   Module['arguments'] = process['argv'].slice(2);
 
+#if MODULARIZE == 0
```

Comment:
please add a comment saying why this is not needed in that mode

Reply:
Comment added